apps/user-office-frontend/src/components/SEP/MeetingComponents/ProposalViewModal/FinalRankingForm.tsx

```diff
@@ -1,355 +1,359 @@
-                          </Box>
-                        )}
-                        {isUserOfficer && (
-                          <Field
-                            id="submitted"
-                            name="submitted"
-                            component={CheckboxWithLabel}
-                            type="checkbox"
-                            Label={{
-                              label: 'Submitted',
-                            }}
-                            disabled={isSubmitting}
-                            data-cy="is-sep-meeting-submitted"
-                          />
-                        )}
-                        <Button
-                          type="submit"
-                          onClick={() => {
-                            setShouldClose(false);
-                            setShouldSubmit(false);
-                          }}
-                          color={isUserOfficer ? 'primary' : 'secondary'}
-                          className={classes.button}
-                          data-cy="save"
-                          disabled={shouldDisableForm(isSubmitting)}
-                        >
-                          Save
-                        </Button>
-                        <Button
-                          type="submit"
-                          onClick={() => {
-                            setShouldClose(true);
-                            setShouldSubmit(false);
-                          }}
-                          color={isUserOfficer ? 'primary' : 'secondary'}
-                          className={classes.button}
-                          data-cy="saveAndContinue"
-                          disabled={shouldDisableForm(isSubmitting)}
-                        >
-                          Save and continue
-                        </Button>
-                        {!isUserOfficer && (
-                          <Button
-                            type="submit"
-                            onClick={() => {
-                              setShouldClose(false);
-                              setShouldSubmit(true);
-                            }}
-                            className={classes.button}
-                            data-cy="submitSepMeeting"
-                            disabled={shouldDisableForm(isSubmitting)}
-                          >
-                            Submit
-                          </Button>
-                        )}
-                      </>
-                    )}
-                  </StyledButtonContainer>
-                </Grid>
-              </Grid>
-            </Form>
-          )}
-        </Formik>
-      </StyledPaper>
-    </div>
-  );
-};
-
-FinalRankingForm.propTypes = {
-  closeModal: PropTypes.func.isRequired,
-  proposalData: PropTypes.any.isRequired,
-  meetingSubmitted: PropTypes.func.isRequired,
-  hasWriteAccess: PropTypes.bool.isRequired,
-};
-
-export default withConfirm(FinalRankingForm);
+import Box from '@mui/material/Box';

```

Comment:
I guess the line ending got screwed up on this file somehow. No changes needed but I wonder how that happened not